OnBoardMediaControl.pro

```diff
@@ -0,0 +1,11 @@
+TEMPLATE = subdirs
+
+CONFIG += ordered static
+
+message(University of Calgary Solar Car Team Epsilon Onboard Media Control)
+
+SUBDIRS = \
+    DataLayer \
+    BusinessLayer \
+    ViewLayer \
+    OnboardMediaControl
```

Comment:
This was the culprit in that error. Since the CONFIG is an `ordered static`, that means each directory is compiled in the order that is given. Before, we had `ViewLayer` first, so when it tried to compile, it couldn't find the files for businesslayer stuff it depended on, since it hadn't been compiled at that time.